.gitignore

```diff
@@ -4,3 +4,5 @@
 .Ruserdata
 ggeasy.Rproj
 inst/doc
+NAMESPACE
```

Comment:
I suspect you don't mean for this to be ignored.

Reply:
it was pushed anyways... ignore only works on nontracked files